### PR TITLE
UNI-11984 - Lens Flare docs

### PR DESCRIPTION
**Pull request by JordanL8**
### Purpose of this PR
Work for https://unity3d.atlassian.net/browse/UNI-11984. Consisted of changing file names and links to adhere to naming conventions, adding some workflow information for creating a lens flare, and adding missing links to the toc.

### CODE REVIEW

**Review by erikabar — APPROVED** [review]
Reviewed doc changes, looks good.

File: com.unity.render-pipelines.high-definition/Documentation~/shared/lens-flare/lens-flare-component.md [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
+![](../../images/shared/lens-flare/lens-flare-comp.png)
 
-![](images/LensFlareComp.png)
+## Creating lens flares in SRP
+
+The Lens Flare (SRP) component controls where the lens flare is as well as properties such as attenuation and whether the lens flare considers occlusion. For properties that define how the lens flare looks, SRP uses the [Lens Flare (SRP) Data](lens-flare-asset.md) asset. Each Lens Flare (SRP) component must reference a Lens Flare (SRP) data asset to display a lens flare on-screen.
+
+To create a lens flare in a scene:
+
+1. Create or select a GameObject to attach the lens flare too.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/shared/lens-flare/lens-flare-component.md:15]: "Too" :D Should be "to".

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Found a teeny tiny typo!

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
adressed pr feedback, merging

**Pull request merged**